lib/cors-anywhere.js

```diff
@@ -293,6 +294,10 @@ var getHandler = exports.getHandler = function(options, proxy) {
       delete req.headers[header];
     });
 
+    for (var header in corsAnywhere.addHeaders) {
+      console.log(header, corsAnywhere.addHeaders[header]);
+      req.headers[header] = corsAnywhere.addHeaders[header];
+    };
```

Comment:
The `{ and }` are part of the `for` block (which is not an expression), so the semicolon is not needed.